Library/Formula/fontforge.rb

```diff
@@ -85,8 +85,12 @@ def install
     ENV["ARCHFLAGS"] = "-arch #{MacOS.preferred_arch}"
 
     # Bootstrap in every build: https://github.com/fontforge/fontforge/issues/1806
+    # "--skip-git" tells the bootstrap script not to pull the latest master
+    # branch of gnulib and just use the commit we've checked out.
     resource("gnulib").fetch
-    system "./bootstrap", "--gnulib-srcdir=#{resource("gnulib").cached_download}"
+    system "./bootstrap",
+      "--gnulib-srcdir=#{resource("gnulib").cached_download}",
+      "--skip-git"
```

Comment:
I think you're fine to wedge this onto the same line personally, but if not let's line up the arguments with the start of `./bootstrap` if possible.